src/main/java/com/jwplayer/jwplatform/utils/Util.java

```diff
@@ -0,0 +1,77 @@
+package com.jwplayer.jwplatform.utils;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.util.Date;
+import java.util.concurrent.ThreadLocalRandom;
+
+import org.apache.commons.lang3.StringUtils;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import com.jwplayer.jwplatform.exception.JWPlatformException;
+import com.jwplayer.jwplatform.exception.JWPlatformUnknownException;
+import com.jwplayer.jwplatform.exception.MediaAPIExceptionFactory;
+
+public class Util {
+	/**
+	   * URL encodes a {@code String}, then modifies it to be compliant with the JW Platform API.
+	   *
+	   * @param stringToEncode - the {@code String} to be URL Encoded
+	   * @return - JW Platform API compliant encoded {@code String}
+	   * @throws JWPlatformException - an exception occurred trying to encode the requested
+	   *     {@code String}
+	   */
+	  public static String encodeStringForJWPlatformAPI(final String stringToEncode)
+	          throws JWPlatformException {
+	    try {
+	      final String encodedValue = URLEncoder.encode(stringToEncode, "utf-8");
+
+	      // string replacements to align with the API
+	      return encodedValue.replace("%7E", "~").replace("*", "%2A").replace("+", "%20");
+	    } catch (final UnsupportedEncodingException e) {
+	      throw new JWPlatformUnknownException(
+	              String.format("Exception thrown encoding URL parameter %s", e.toString()));
+	    }
+	  }
+	  
+	  /**
+	   * Check if the API response is an non-200. If so, throw the
+	   * appropriate JWPlatformException exception based on the
+	   * error message.
+	   *
+	   * @param statusCode - the response status code
+	   * @param response - a {@code JSONObject} object with the API response block
+	   * @throws JWPlatformException - API returned an exception
+	   */
+	  public static void checkForNon200Response(final int statusCode, final JSONObject response)
+	          throws JWPlatformException {
+	    if (statusCode != 200 && statusCode != 201) {
```

Comment:
Do we just want to check the first digit is 2? Not sure if there are other possible 200 response codes from the api.